src/components/molecules/CoreTable/CoreTable.tsx

```diff
@@ -0,0 +1,231 @@
+    return 'bg-red-100 text-error-700';
+};
+
+const CollateralRatioBadge = ({ ratio }: CollateralRatioBadgeProps) => {
+    return (
+        <span
+            className={clsx(
+                'inline-block rounded-full px-2 py-1 text-xs font-medium',
+                getCollateralRatioColor(ratio)
+            )}
+        >
+            {new Percent(ratio).prettify()}
+        </span>
+    );
+};
+
+const TroveRow = ({
+    trove,
+    price,
+    isConnected,
+    sfStablecoin,
+    open,
+    index,
+    currentTxId,
+    setCurrentTxId,
+}: TroveRowProps) => {
+    const [copied, setCopied] = useState<string | undefined>();
+
+    const txId = `liquidate-${trove.ownerAddress}-${index}`;
+
+    useEffect(() => {
+        if (copied !== undefined) {
+            let cancelled = false;
+
+            navigator.clipboard.writeText(copied);
+
+            setTimeout(() => {
+                if (!cancelled) {
+                    setCopied(undefined);
+                }
+            }, 2000);
+
+            return () => {
+                cancelled = true;
+            };
+        }
+    }, [copied]);
+
+    const calculateCollateralRatio = (trove: UserTrove) => {
+        return trove.collateralRatio(price);
+    };
+
+    return (
+        <tr
+            className={clsx(
+                'border-t border-[#f0f0f0]',
+                index % 2 === 0 ? 'bg-[#FAFAFA]' : 'bg-[#FFFFFF]'
+            )}
+        >
+            <td className='p-4'>
+                <div className='flex items-center'>
+                    <span>{AddressUtils.format(trove.ownerAddress, 6)}</span>
+                    <button
+                        className='ml-2 text-[#565656] hover:text-primary-500'
+                        onClick={() => setCopied(trove.ownerAddress)}
+                    >
+                        {copied === trove.ownerAddress ? (
+                            <CheckIcon size={12} className='text-success-700' />
+                        ) : (
+                            <Copy size={12} />
+                        )}
+                    </button>
+                </div>
+            </td>
+            <td className='p-4'>{trove.collateral.prettify()}</td>
+            <td className='p-4'>{trove.debt.prettify()}</td>
+            <td className='p-4'>
+                <CollateralRatioBadge ratio={calculateCollateralRatio(trove)} />
+            </td>
+            <td className='p-4'>{trove.debtInFront?.prettify()}</td>
+
+            <td className='px-2 py-4'>
+                <div className='items-end laptop:w-56'>
+                    {index === 0 && !isConnected ? (
+                        <button
+                            onClick={open}
+                            className='truncate rounded-xl border border-[#E3E3E3] bg-primary-500 px-4 py-2 text-sm text-[#fff] laptop:w-56'
+                        >
+                            Connect wallet
+                        </button>
+                    ) : (
```

Comment:
Why does the first button always become "Connect Wallet".
It seems this address is my wallet or something but not, so it looks weird.

<img width="1683" height="935" alt="USDFC__FIL-Backed_Stablecoin___Secured_Finance" src="https://github.com/user-attachments/assets/20b196e0-6464-4dec-b0c6-b43718a05dba" />